### DISCUSSION TITLE
Is it possible to make the app's background more transparent?

### DISCUSSION THREAD

**Discussion by christina-de-martinez** [callstack/react-native-visionos#124] [Q&A] [answered]
Hi! This is a beginner question, and I hope I'm in the right spot to ask. Please feel free to redirect me otherwise. 

I'm wondering if there's a way to change the semi-transparent gray background that comes with the visionOS starter app? I'd like to make it more transparent so that my app blends in with the environment more. I see that I could edit the background color with CSS if I wanted to make it darker for whatever reason, but I'm wanting to go the other way. Maybe this is just impossible to do or ill-advised, but I thought I'd ask. Thanks in advance for the help!

**Comment by okwasniewski** [comment, marked as answer]
Hey! 

Thanks for asking this question, this made me wonder how is it possible that React Native visionOS apps are a little bit darker than SwiftUI apps and I've found the root cause (we had an additional color set on the background). 

### Before 

(SwiftUI on the left, React Native on the right)

![CleanShot 2024-02-29 at 09 38 07@2x](https://github.com/callstack/react-native-visionos/assets/52801365/4ac4db9a-2209-4802-b8cc-9374b8881648)

### After

![CleanShot 2024-02-29 at 09 38 40@2x](https://github.com/callstack/react-native-visionos/assets/52801365/30b4f4b0-7304-421a-a2fa-7ea0fa6dadc8)

I'll create a PR with the fix soon 👍🏻

**Reply by christina-de-martinez** [reply]
Thanks so much!! This looks more like what I'd want.

**Reply by okwasniewski** [reply]
Released in 0.73.7 🚀

**Comment by okwasniewski** [comment]
https://github.com/callstack/react-native-visionos/pull/125
